### PR TITLE
Map: Youtube video

### PR DESCRIPTION
**Pull request by RubenMattan**

### CODE REVIEW

File: rogerthat/build.gradle [hunk 1]
```diff
@@ -162,6 +162,7 @@ dependencies {
     implementation 'com.tbuonomo.andrui:viewpagerdotsindicator:4.1.2'
     implementation 'com.github.ialokim:android-phone-field:0.2.3'
     implementation 'com.github.Ghitodin:floatingsearchview:2.2.3'
+    implementation 'com.r0adkll:slidableactivity:2.1.0'
```
Comment (lucasvanhalst) [inline rogerthat/build.gradle:165]: moet weg

File: rogerthat/src/main/java/com/mobicage/rogerthat/plugins/maps/MediaSectionAdapter.kt [hunk 2]
```diff
@@ -24,32 +32,58 @@ class MediaSectionAdapter : PagerAdapter() {
...
+                        "media" to section.toJSONMap()
+                )
+                val ctx = OpenEmbeddedAppContext(OpenEmbeddedAppContextType.MEDIA_360_VIEWER, contextDict)
+                val extras = Bundle()
+                extras.putString(ActionScreenActivity.CONTEXT, JSONValue.toJSONString(ctx.toJSONMap()))
+                extras.putString(CordovaActionScreenActivity.EMBEDDED_APP_ID, "media-viewer")
+
+                val i = Intent(context, CordovaActionScreenActivity::class.java)
+                i.putExtras(extras)
+                i.addFlags(MainActivity.FLAG_CLEAR_STACK)
```
Comment (RubenMattan) [inline rogerthat/src/main/java/com/mobicage/rogerthat/plugins/maps/MediaSectionAdapter.kt:77]: why clear stack?

File: rogerthat/src/main/java/com/mobicage/rogerthat/plugins/maps/MediaSectionAdapter.kt [hunk 3]
```diff
@@ -24,32 +32,58 @@ class MediaSectionAdapter : PagerAdapter() {
...
+            NewsStreamAdapter.MEDIA_TYPE_IMAGE_360, NewsStreamAdapter.MEDIA_TYPE_VIDEO_360 -> {
+                // semi-crappy solution using an embedded app to load a 360 viewer because
+                // up-to-date documentation for native implementation is just nowhere to be found
+                val contextDict = mapOf(
+                        "media" to section.toJSONMap()
+                )
+                val ctx = OpenEmbeddedAppContext(OpenEmbeddedAppContextType.MEDIA_360_VIEWER, contextDict)
+                val extras = Bundle()
+                extras.putString(ActionScreenActivity.CONTEXT, JSONValue.toJSONString(ctx.toJSONMap()))
+                extras.putString(CordovaActionScreenActivity.EMBEDDED_APP_ID, "media-viewer")
```
Comment (RubenMattan) [inline rogerthat/src/main/java/com/mobicage/rogerthat/plugins/maps/MediaSectionAdapter.kt:73]: rename to media-360-viewer

**Comment by RubenMattan** [comment]
LGTM

**Pull request merged**